assets/blocks/quiz/category-question-block/category-question-edit.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * WordPress dependencies
+ */
+import { __, _n, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { useQuestionIndex } from '../question-index';
+import { useQuestionCategories } from '../question-categories';
+import CategoryQuestionSettings from './category-question-settings';
+
+/**
+ * Quiz category question block editor.
+ *
+ * @param {Object}   props
+ * @param {Object}   props.attributes              Block attributes.
+ * @param {Object}   props.attributes.categoryName Category name.
+ * @param {Function} props.setAttributes           Set block attributes.
+ */
+const CategoryQuestionEdit = ( props ) => {
+	const {
+		attributes: {
+			options: { number = 1, category },
+		},
+		clientId,
+	} = props;
+	const index = useQuestionIndex( clientId );
+	const [ , getCategoryTermById ] = useQuestionCategories();
+
+	const nextNumber = index;
+	let range = nextNumber;
+	if ( number !== 1 ) {
+		range += ' - ' + ( nextNumber + number - 1 );
```

Comment:
A bit scared of the integer + string operation here, thought it seems to be working correctly. Might be safer with a template string:

```suggestion
	const range = 1 === number ? index : `${ index } - ${ index  + number - 1}`;
```

Reply:
Smart thinkin' 6eca124f0597be9e48d449a723a9cc8d47cc61dd